main.tf

```diff
@@ -10,6 +10,8 @@ resource "random_id" "id" {
 resource "cloudfoundry_service_instance" "redis" {
   name  = "tf-redis-${local.postfix}"
   space = var.cf_space_id
+  tags                           = var.tags
```

Comment:
The tags does not seem to have a default making this param mandatory? Would be nice to have a sane default.

Reply:
i have given default value for tags as empty list